src/ServicePulse.Host/vue/src/composables/formatter.ts

```diff
@@ -35,22 +35,22 @@ export function useGetDayDiffFromToday(value: string) {
   return Math.round(diff / 1000 / 60 / 60 / 24);
 }
 
-export function useFormatLargeNumber(value: string, decimals: number) {
+export function useFormatLargeNumber(value: number, decimals: number) {
   const suffixes = ["k", "M", "G", "T", "P", "E"];
 
   const num = Number(value);
 
   if (isNaN(num)) {
```

Comment:
the type says is a number, do we need this?

Reply:
NaN is a valid number